samples/java-eventsourced-counter-pojo/src/main/java/org/example/domain/ValueReset.java

```diff
@@ -0,0 +1,13 @@
+package org.example.domain;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+@JsonTypeName("reset")
```

Comment:
is this needed?

Reply:
Yes, it's needed if we use an object. Since there is now data, the payload is serialized to `{}`. When it comes back we need to know how to deserialize it back, so we need a type hint. 

But maybe not needed if we use enums.